txpipe/input_cats.py

```diff
@@ -28,12 +29,14 @@ class TXCosmoDC2Mock(PipelineStage):
     config_options = {
         'cat_name':'cosmoDC2',
         'visits_per_band':165,  # used in the noise simulation
-        'snr_limit':4.0,  # used to decide what objects to cut out
+        'snr_limit': 4.0,  # used to decide what objects to cut out
+        'Mag_r_limit': -19, # used to decide what objects to cut out  
```

Comment:
could you move this down so it's next to apply_mag_cut so that when I see it I'll realise it's not always applied?